### ISSUE TITLE
sidebars shrink on resolution change since 1.70.0

### ISSUE THREAD

**Issue by af4jm** [microsoft/vscode#157902]
Type: <b>Bug</b>

In all versions up to & including 1.69.x, as I opened & closed my PC and connected & disconnected my external monitor, the sidebars maintained the same width. Since 1.70.0 (and still today in 1.70.1) they shrink so narrow that I get only the first character of files with git changes and no icons at all in extensions (my typical width is the narrowest width that allows large icons in extensions).

The overall window size does not change, nor does the height of the panel.

The narrowing only occurs when the screens change in Windows. Moving the window between different resolution screens maintains the same sidebar width, as does exiting & restarting VSCode.

In case it's relevant, my notebook PC screen is 17" 1600x900, my external is 32" 2560x1440

VS Code version: Code 1.70.1 (6d9b74a70ca9c7733b29f0456fd8195364076dda, 2022-08-10T06:08:33.642Z)
OS version: Windows_NT x64 10.0.22000
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8265U CPU @ 1.60GHz (8 x 1800)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.89GB (18.88GB free)|
|Process Argv|--file-uri file:///c%3A/src/AF4JM.code-workspace --crash-reporter-id 06338080-eec2-498d-98e0-cf3111dc1cde|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (61)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-icalendar|af4|1.0.1
vscode-m3u|af4|1.0.0
vscode-caniuse|aga|0.5.0
case-change|Aka|1.0.2
rtf|ale|2.5.1
spellright|ban|3.0.72
markdown-checkbox|bie|0.3.2
markdown-emoji|bie|0.3.0
markdown-footnotes|bie|0.0.7
markdown-mermaid|bie|1.14.3
markdown-yaml-preamble|bie|0.1.0
vscode-tldr|bmu|1.0.0
mermaid-markdown-syntax-highlighting|bpr|1.3.4
npm-intellisense|chr|1.4.2
disableligatures|Coe|0.0.10
vscode-markdownlint|Dav|0.47.0
jshint|dba|0.11.0
vscode-eslint|dba|2.2.6
rushcore|Dev|1.0.2
rushnav|Dev|1.0.2
githistory|don|0.6.19
xml|Dot|2.5.1
escaping-characters|drp|1.0.0
EditorConfig|Edi|0.16.4
vscode-macros|EXC|1.4.0
vscode-npm|fkn|3.3.0
shell-format|fox|7.2.2
macros|ged|1.2.1
matlab|Gim|2.3.1
gc-excelviewer|Gra|4.2.55
vscode-favorites|how|1.10.2
rest-client|hum|0.25.0
path-autocomplete|ion|1.20.2
mediawiki|jak|2.1.0
anki|jas|1.2.6
markdown-katex|jef|0.1.4
nuget-reverse-package-search|jes|0.1.68
vscode-nuget-package-manager|jmr|1.1.6
vscode-peacock|joh|4.0.1
gpg|jva|0.1.2
docomment|k--|0.1.31
markdown-shortcuts|mdi|0.12.0
azure-pipelines|ms-|1.208.0
vscode-azureappservice|ms-|0.24.3
vscode-azureresourcegroups|ms-|0.5.5
vscode-azurestaticwebapps|ms-|0.11.2
vscode-azurestorage|ms-|0.14.2
csharp|ms-|1.25.0
vscode-dotnet-runtime|ms-|1.5.0
python|ms-|2022.12.0
vscode-pylance|ms-|2022.8.20
azure-account|ms-|0.11.1
hexeditor|ms-|1.9.8
powershell|ms-|2022.7.2
vscode-node-azure-pack|ms-|1.0.0
bib|phr|0.3.0
vscode-yaml|red|1.9.1
vscode-zipexplorer|sle|0.3.1
addDocComments|ste|0.0.8
vscodeintellicode|Vis|1.2.22
vscode-icons|vsc|11.15.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30542924
vsaa593cf:30376535
pythonvs932:30410667
wslgetstarted:30449410
cppdebug:30492333
pylanb8912cf:30529770
vsclangdc:30486549
c4g48928:30535728
hb751961:30543456
dsvsc012cf:30540253

```

</details>

**Comment by sbatten** [comment]
@af4jm can you clarify the exact repro steps? e.g. where the window is at each step.

**Comment by af4jm** [comment]
I think it was fixed in 1.71 because I haven't noticed it in a while... but it was when the maximized window changed size because of a screen resolution change (in my case, flipping the AB switch on my docking station between my PC & the work computer)

**Comment by sbatten** [comment]
thanks @af4jm closing in light of this
